export2.js

```diff
@@ -64,7 +64,7 @@ function exportToCsv(filename, rows) {
             navigator.msSaveBlob(blob, filename);
         } else {
             var link = document.createElement("a");
-            if (link.download !== undefined) { // feature detection
+            if (link.download !== "undefined") { // feature detection

```

Comment:
Did you test this? This is comparison straight to a property, not to the result of "typeof", sure this is supposed to be a string?